.github/scripts/get-release-branches.js

```diff
@@ -20,17 +20,20 @@ module.exports = ({ github, core }) => {
             issueCommentBody = issue.body_text
             if (issueCommentBody.includes("#Release#")) {
                 let components = issueCommentBody.split("\n")
-                components = components.splice(2, components.length - 1)
+                const releaseIdx = components.indexOf("#Release#")
+                components = components.splice(releaseIdx + 1, components.length - 1)
                 components.forEach(component => {
-                    [componentName, branchUrl] = component.split("|")
-                    const splitArr = branchUrl.split("/")
-                    const idx = splitArr.indexOf("tree")
-                    const branchName = splitArr.slice(idx + 1).join("/")
-                    if(componentName === "notebook-controller"){
-                        core.exportVariable("component_spec_odh-notebook-controller".toLowerCase(), branchName);
-                        core.exportVariable("component_spec_kf-notebook-controller".toLowerCase(), branchName);
-                    }else{
-                        core.exportVariable("component_spec_"+componentName.toLowerCase(), branchName);
+                    if (component) {
+                        [componentName, branchUrl] = component.split("|")
+                        const splitArr = branchUrl.split("/")
```

Comment:
@AjayJagan can we modify this block using regexp, that would be better here.

Reply:
this is done now